src/main/java/io/r2dbc/postgresql/PostgresqlConnectionConfiguration.java

```diff
@@ -215,6 +225,21 @@ public Builder host(String host) {
             return this;
         }
 
+        /**
+         * Configure runtime options.
+         *
+         * These parameters are applied once after creating a new connection. This is useful for setting up client-specific
+         * <a href="https://www.postgresql.org/docs/current/runtime-config-client.html#RUNTIME-CONFIG-CLIENT-FORMAT">runtime parameters</a>
+         * like statement timeouts, time zones etc.
+         *
+         * @param options the options map
+         * @return this {@link Builder}
+         */
+        public Builder options(@Nullable Map<String, ?> options) {
```

Comment:
How about keeping `options` non-nullable? Reducing nullability helps to prevent potential NPE's in the first place.

Reply:
True. I'll change it.